pkgs/tools/misc/parallel/wrapper.nix

```diff
@@ -2,12 +2,15 @@
 , perlPackages
 , extraPerlPackages ?
     with perlPackages; [ DBI DBDPg DBDSQLite DBDCSV TextCSV ]
+, disableNagMessage ? false
+, lib
```

Comment:
Nit (sorry): `lib` should be the first input (as noted by https://github.com/nixos/nixpkgs/blob/1ce696fb1d0ce011a19bdfdc495f93e442d23feb/doc/contributing/coding-conventions.xml#L183-L184).